dockerfiles/hpcc/templates/dllserver-claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: dllserver-pv-claim
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
```

Comment:
Will this eventually change to ReadWriteMany?

Reply:
Not sure what the difference is...

Reply:
The access modes are: ReadWriteOnce – the volume can be mounted as read-write by a single node. ReadOnlyMany – the volume can be mounted read-only by many nodes. ReadWriteMany – the volume can be mounted as read-write by many nodes.

So yes, it should say ReadWriteMany if we want to go beyond a single node.